CTFtimeDiscordHooks.py

```diff
@@ -52,13 +58,26 @@ def __init__(self, json_obj: dict):
         if self.restrictions is None or self.restrictions == '':
             self.restrictions = 'Unknown'
         self.duration = timedelta(**json_obj.get('duration', dict()))
+        
+        self.fields = [{
+            "name": "Weight",
+            "value": str(json_obj.get('weight', 0.0))
+        },{
+            "name": "Interested teams",
+            "value": str(json_obj.get('participants', 0))
+        }]
+        
+        self.dayOfWeek = CTF.parse_dayOfWeek(self.finish, self.duration)
```

Comment:
please remove this `day of week` feature as we already display the date on the embed using `timestamp=self.start`
this way discord is viewing the time in the reader's timezone and there is no need to do it ourselves

Reply:
Cause I wanna show it more details on my server, it helps me remember what time the event will be started. 😄

Reply:
but this info is already there

![image](https://user-images.githubusercontent.com/14138694/142733731-7f8da87a-936d-4d8d-bb37-42b28022c6f2.png)